multy_test/test_bitcoin_transaction.cpp

```diff
@@ -402,9 +402,9 @@ GTEST_TEST(BitcoinTransactionTest, Unprofitable_change)
     ASSERT_EQ("0", change_amount);
 
     //NOTE: if change address amount == 0 serialize without change address
-    //      TXid: 667b0fb64e1afd75efc8d32508904bbf781335b6e30f84a07cef756bf4260ba9
```

Comment:
IMO, we should remove that serialized TX check here, since we are not testing TX serialization here.